### PR TITLE
python312Packages.opensearch-py: fix build

### PR DESCRIPTION
**Pull request by drupol**
An issue has been opened upstream at https://github.com/opensearch-project/opensearch-py/issues/849

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by drupol** [comment]
@ofborg build python312Packages.opensearch-py

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 357642`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

**Comment by drupol** [comment]
@ofborg build python312Packages.opensearch-py

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 357642`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 357642`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 357642`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/pygelf/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  buildPythonPackage,
+  fetchPypi,
+  mock,
+  pytestCheckHook,
+  requests,
+  cacert,
+}:
+buildPythonPackage rec {
+  pname = "pygelf";
+  version = "0.4.2";
```
Comment (natsukium) [inline pkgs/development/python-modules/pygelf/default.nix:12]:
```suggestion
  version = "0.4.2";
  pyproject = true;
```

File: pkgs/development/python-modules/pygelf/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  src = fetchPypi {
+    pname = "pygelf";
+    inherit version;
+    hash = "sha256-0LuPRf9kipoYdxP0oFwJ9oX8uK3XsEu3Rx8gBxvRGq0=";
+  };
```
Comment (natsukium) [inline pkgs/development/python-modules/pygelf/default.nix:18]:
```suggestion
  };
  
  build-system = [ setuptools ];
```

**Review by SuperSandro2000 — APPROVED** [review]
LGTM

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 357642`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.parsedmarc.dist</li>
    <li>python312Packages.parsedmarc</li>
    <li>python312Packages.parsedmarc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 22 packages built:</summary>
  <ul>
    <li>parsedmarc</li>
    <li>parsedmarc.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.mail-parser</li>
    <li>python311Packages.mail-parser.dist</li>
    <li>python311Packages.mailsuite</li>
    <li>python311Packages.mailsuite.dist</li>
    <li>python311Packages.opensearch-py</li>
    <li>python311Packages.opensearch-py.dist</li>
    <li>python311Packages.pygelf</li>
    <li>python311Packages.pygelf.dist</li>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.mail-parser</li>
    <li>python312Packages.mail-parser.dist</li>
    <li>python312Packages.mailsuite</li>
    <li>python312Packages.mailsuite.dist</li>
    <li>python312Packages.opensearch-py</li>
    <li>python312Packages.opensearch-py.dist</li>
    <li>python312Packages.pygelf</li>
    <li>python312Packages.pygelf.dist</li>
  </ul>
</details>

**Pull request merged**